api/openapi.generated.yml

```diff
@@ -51,7 +51,7 @@ paths:
       tags:
```

Comment:
Is this YAML file autogenerated before it's checked into git? (Sorry I'm not familiar with how our system currently works on this front!)

Reply:
Yes, there is a github action that runs automatically to do it. I just made a few updates based on what Aaron suggested, and it triggered: https://github.com/HHS/simpler-grants-gov/actions/runs/7387946234/job/20097725000?pr=939 which added this commit: https://github.com/HHS/simpler-grants-gov/pull/939/commits/759df0f253f6eeb7488d18cf92d47b1757244bc7

Note, the yml file is not used by the API and is just reference. The file it uses when running is generated when the app starts up (exact same underlying code generates them).